src/docker/Dockerfile

```diff
@@ -56,39 +57,16 @@ RUN apt-get install -y software-properties-common python-software-properties
 RUN add-apt-repository ppa:git-core/ppa
 RUN apt-get update && apt-get install -y git git-core
 
-# SSH Daemon
-RUN apt-get install -y ssh && \
-  mkdir /var/run/sshd && \
-  chmod 0755 /var/run/sshd
-
-# Ruby
-RUN apt-get install -y libmagic-dev zlib1g-dev
-RUN apt-add-repository ppa:brightbox/ruby-ng
-RUN apt-get update
-RUN apt-get install -y ruby2.2 ruby2.2-dev
-RUN gem update
-RUN gem install nokogiri
-RUN gem install bundler
-
 # Java
-RUN apt-get install -y default-jdk
+RUN apt-get install -y openjdk-7-jdk="${OPENJDK_VERSION}"
```

Comment:
@amihaiemil while you're at it, please make this `apt-get update && apt-get install -y ...`, see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#/apt-get for an explanation ;)